keps/sig-contributor-experience/0034-artifact-management.md

```diff
@@ -0,0 +1,125 @@
+---
+kep-number: 34
+title: Kubernetes Community Artifact Serving
+authors:
+  - "@brendandburns"
+owning-sig: sig-contributor-experience
```

Comment:
Is this the right SIG? if so we should CC them
I might suggest release as a possible better fit (or maybe testing)?

Reply:
contrib-ex or cluster-lifecycle, IMO

Reply:
ok, I'm sticking w/ contrib-ex unless there are violent objections.